pkg/continuoustest/util_test.go

```diff
@@ -141,6 +142,13 @@ func testVerifySamplesSumHistograms(t *testing.T, generateValue generateValueFun
 	// Round to millis since that's the precision of Prometheus timestamps.
 	now := time.UnixMilli(time.Now().UnixMilli()).UTC()
```

Comment:
Not part of this change but this can be:
```suggestion
	now := time.Now().Round(time.Millisecond)
```